streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/kstream/KTable.scala

```diff
@@ -227,9 +225,9 @@ class KTable[K, V](val inner: KTableJ[K, V]) {
    * one for each matched record-pair with the same key
    * @see `org.apache.kafka.streams.kstream.KTable#leftJoin`
    */ 
-  def leftJoin[VO, VR](other: KTable[K, VO],
-    joiner: (V, VO) => VR,
-    materialized: Materialized[K, VR, ByteArrayKeyValueStore]): KTable[K, VR] =
+  def leftJoin[VO, VR](other: KTable[K, VO], materialized: Materialized[K, VR, ByteArrayKeyValueStore])(
```

Comment:
Ditto here? Also the parameters are re-ordered.

Reply:
Indeed I forgot this one